[
 {
  "author": "bodewig", 
  "date": "2011-11-17T11:56:43.179963Z", 
  "msg": [
   "no reason to log at ERR if a URL doesn't exist and all we do is checking whether it does.  Addresses PR 51802"
  ], 
  "revision_id": "1203160"
 }, 
 {
  "author": "rmannibucau", 
  "date": "2011-11-17T11:34:52.930836Z", 
  "msg": [
   "OPENEJB-1063 OPENEJB-1708 fixing the client main class when it contains a ."
  ], 
  "revision_id": "1203159"
 }, 
 {
  "author": "julianfoad", 
  "date": "2011-11-17T11:31:57.620731Z", 
  "msg": [
   "On the 'showing-merge-info' branch: Fix up tests so that most of them pass.", 
   "* subversion/tests/cmdline/input_validation_tests.py", 
   "(run_and_verify_svn_in_wc): Tolerate messages on stdout before the", 
   "expected error.", 
   "* subversion/tests/cmdline/merge_reintegrate_tests.py", 
   "(reintegrate_with_subtree_mergeinfo): Same.", 
   "* subversion/tests/cmdline/mergeinfo_tests.py", 
   "(no_mergeinfo): Expect a \"Source and target are the same branch\" error.", 
   "* subversion/tests/cmdline/merge_tests.py", 
   "(expected_merge_output): Expect the 'Sync|Reint|... merge from ...' header.", 
   "(simple_property_merges, mergeinfo_inheritance): Tolerate messages on", 
   "stdout for a no-op merge.", 
   "(dont_merge_revs_into_subtree_that_predate_it,", 
   "merge_with_os_deleted_subtrees): Tolerate messages on stdout before the", 
   "expected error.", 
   "* subversion/tests/cmdline/svntest/actions.py", 
   "(expected_noop_merge_output): New function.", 
   "(run_and_verify_merge): Strip off the 'This is a dry-run merge ...' header", 
   "lines. Expect a 'To continue using ...' message at the end of a", 
   "reintegate merge. Tweak the diagnostic output.", 
   "(run_and_verify_mergeinfo): Default to '--show-revs=eligible' mode, making", 
   "the new default mode inaccessible, for now."
  ], 
  "revision_id": "1203158"
 }, 
 {
  "author": "julianfoad", 
  "date": "2011-11-17T11:31:16.716688Z", 
  "msg": [
   "On the 'showing-merge-info' branch: Don't print the keep-alive message if", 
   "the merge failed.", 
   "* subversion/svn/merge-cmd.c", 
   "(svn_cl__merge): As above."
  ], 
  "revision_id": "1203157"
 }, 
 {
  "author": "julianfoad", 
  "date": "2011-11-17T11:25:07.883150Z", 
  "msg": [
   "On the 'showing-merge-info' branch: Catch up to trunk@1203154."
  ], 
  "revision_id": "1203156"
 }
]